galley/pkg/runtime/flow/doc.go

```diff
@@ -0,0 +1,59 @@
+//  Copyright 2018 Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+// Package flow contains primitive building blocks for creating data processing pipelines for conversion
+// purposes. The core abstractions are:
```

Comment:
What other resources (apart from the unused ingress) do you envision using this pipeline? The service registry conversion has a different structure. Other networking resources are unit resources, that dont generate anything. Ingress happens to be the only exception.

Reply:
endpoints will be the other one, in the immediate term.